package.json

```diff
@@ -47,6 +47,15 @@
         "category": "ONE",
         "_comment": "Call this when compiling compilable model(e.g., *.tflite, *.onnx, etc.)"
       },
+      {
+        "command": "onevscode.refresh-compiler",
+        "title": "Refresh",
```

Comment:
Compile-Refresh?

Reply:
The above command `onevscode.show-compile-webview`'s title is "Compile".

Reply:
Refresh-Compile?

Reply:
AFAIK, title will be shown to user. (may as a tooltip in case of icon?). So, human readable format would be better. 
I like current one (`Refresh`) or `Refresh Compiler Toolchains`.

Reply:
This title will be shown as a tooltip. And this icon is already attached to Compiler Toolchain. So I think it doesn't need to mention `Compiler` again.